app/app.jsx

```diff
@@ -2,14 +2,27 @@ import React from 'react';
 import './scss/style.scss';
 import 'normalize.css';
 
+/* import components */
+import Main from './containers/main.jsx';
+import Nav from './containers/nav.jsx';
+
 class App extends React.Component {
+
    render() {
+
+     const store = this.props.store;
```

Comment:
Ok this is not the right way.

Take 10 mins to read this: http://redux.js.org/docs/basics/UsageWithReact.html

Feel free to ask me any questions!

Basically you app should be a container..